tests/utils/inputparams/testparams.go

```diff
@@ -43,10 +51,14 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
```

Comment:
We should remove this and use GetUniqueVolumeName instead in all tests. I've filed issue #1344 to track this separately.